cvat/apps/engine/task.py

```diff
@@ -577,7 +578,11 @@ def _find_and_compress_images(upload_dir, output_dir, db_task, compress_quality,
             job.meta['status'] = 'Images are being compressed.. {}%'.format(idx * 100 // len(filenames))
             job.save_meta()
             compressed_name = os.path.splitext(name)[0] + '.jpg'
-            image = Image.open(name).convert('RGB')
+            image = Image.open(name)
+            if image.mode == "I":
```

Comment:
Hi @Happyzippy , thanks for your contribution! Could you please add comment for the change? It a known issue with PIL as I can see from here: https://github.com/python-pillow/Pillow/issues/2574. It will be difficult to understand the change in the future without any comments.

Reply:
Hi @nmanovic. I have added some comments and made the change a bit more readable